src/components/nav/nav-item-dropdown.js

```diff
@@ -29,10 +29,6 @@ export const BNavItemDropdown = /*#__PURE__*/ Vue.extend({
     toggleId() {
       return this.safeId('_BV_toggle_')
     },
-    isNav() {
```

Comment:
This was a leftover from https://github.com/bootstrap-vue/bootstrap-vue/pull/4426.